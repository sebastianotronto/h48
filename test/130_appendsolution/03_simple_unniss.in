U F B2 (R' F D)
1
1
rotation UF
0
