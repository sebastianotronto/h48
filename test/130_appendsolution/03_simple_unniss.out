U F B2 D' F' R
Number of solutions: 1
Shortest solution length: 6
Used bytes: 15
